Documentation/ceph-csi-troubleshooting.md

```diff
@@ -441,3 +441,54 @@ $ rbd ls --id=csi-rbd-node -m=10.111.136.166:6789 --key=AQDpIQhg+v83EhAAgLboWIbl
 ```
 
 Where `-m` is one of the mon endpoints and the `--key` is the key used by the CSI driver for accessing the Ceph cluster.
+
+## Node Loss
+
+When a node is lost, you will see application pods on the node stuck in the `Terminating` state while another pod is rescheduled and is in the `ContainerCreating` state.
+
+To allow the application pod to start on another node, Kubernetes requires you to force delete the pod. After a timeout (8-10 minutes), your pod will be allowed to start on another node.
+### Force delete your application pod. After some timeout (8-10 minutes), your pod will be rescheduled
+
+To force delete the pod stuck in the `Terminating` state:
+
+```console
+$ kubectl -n rook-ceph delete pod my-app-69cd495f9b-nl6hf --grace-period 0 --force
+```
+
+**And** then wait for timeout (8-10 minutes).
+
+### Shorten the timeout
+
+To shorten the timeout, you can mark the node as "blacklisted" so Rook can safely failover the pod sooner
+
+The solution is to remove the watcher, following the commands below from the [Rook toolbox](ceph-toolbox.md):
+
+```console
+kubectl -n rook-ceph exec deploy/rook-ceph-tools -- bash
+```
+```console
+rbd ls --pool=replicapool
+```
+>```
+> csi-vol-1a1bbb34-7461-11eb-a32b-0a580a81020c
```

Comment:
please specify how to identify the image which we need to check

Reply:
added.